tests/OmniSharp.Roslyn.CSharp.Tests/OmniSharpWorkspaceFacts.cs

```diff
@@ -0,0 +1,145 @@
+﻿using Microsoft.CodeAnalysis;
+using Microsoft.Extensions.Logging;
+using OmniSharp.FileWatching;
+using OmniSharp.Services;
+using System.Linq;
+using Xunit;
```

Comment:
Nit: The OmniSharp code base sorts using directives with `System` namespace on top.

Reply:
GitHub diff is being odd. This is fixed though if you look at the FilesChanged